### PR TITLE
Post: Apt Is A Who Not A What

### PR DESCRIPTION
**Pull request by sroberts**

### CODE REVIEW

**Comment by krmaxwell** [comment]
Important post! I'd try to find some references for the original use of APT. I'm pretty sure there are some public references. In fact I've heard some of the folks who were on the original task force that came up with the team say this exact thing in public fora. 

Mind, I think this is a lost cause at this point, but that doesn't change the fact that you're right.

image

File: _posts/2015-02-16-apt-is-a-who-not-a-what.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This was a weird time, because like today attacks weren't just aimed at military targets but commercial targets as well. AFOSI, as well as other government intelligence & defense groups, were investigating using open source and classified methods. For operational security this leads to classified intelligence.
+
+This classified intelligence was a problem. While companies in the Defense Industrial Base may have people with clearances who can handle classified data other types of companies, without cleared people who could handle that intelligence, were under attack as well. These DoD/IC teams wanted to help, but couldn't disclose classified information.  They came up with a compromise: __Sharing indicators and information without disclosing the actual actor behind it. Instead they just referred to the actor using a pseudonym: The Advanced Persistent Threat.__ Specifically APT was a couple groups of actors primarily operating out of mainland China and believed to be members of the People's Liberation Army. We now know these groups today as _APT1_, _Anchor Panda_, and _Elderwood_ as well as other private designations.
```
Comment (ivanlei) [inline _posts/2015-02-16-apt-is-a-who-not-a-what.md]: _While companies in the Defense Industrial Base may have people with clearances who can handle classified data other types of companies, without cleared people who could handle that intelligence, were under attack as well._

I find this hard to parse.

Comment (sroberts) [inline _posts/2015-02-16-apt-is-a-who-not-a-what.md]: Thanks @ivanlei. I agree I haven't loved that sentence. I'll rework it.

**Comment by sroberts** [comment]
Also included feedback from @krmaxwell in https://github.com/sroberts/sroberts.github.io/pull/95
